docs/api_refs/scripts/create-entrypoints.js

```diff
@@ -14,14 +15,14 @@ const updateJsonFile = (relativePath, updateFunction) => {
 
 function main() {
   const project = new Project();
+  const workspaces = fs
+  .readdirSync("../../libs/")
+  .filter((dir) => dir.startsWith("langchain-"))
+  .map((dir) => path.join("../../libs/", dir, "/scripts/create-entrypoints.js"));
   const entrypointFiles = [
     "../../langchain/scripts/create-entrypoints.js",
     "../../langchain-core/scripts/create-entrypoints.js",
```

Comment:
We should probably move `langchain-core` and `langchain` into `libs` to match Python haha